_docs/sql-reference/sql-commands/009-analyze-table-compute-statistics.md

```diff
@@ -99,7 +99,7 @@ Controls the 'compression' factor for the TDigest algorithm used for histogram s
 
 ## Reserved Keywords
 
-The ANALYZE TABLE statement introduces the following reserved keywords:  
+The ANALYZE TABLE COMPUTE STATISTICS statement introduces the following reserved keywords:  
```

Comment:
(IMHO, adding two long words to the command did not help usability.)

Reply:
Command wasn't changed, just added additional words which already are part of the command to distinguish it from another command which starts similarly.